src/main.c

```diff
@@ -85,6 +87,7 @@ int get_key(void) {
 void press_exit(void) {
 	printf("Press any key to exit this application.\n");
 	get_key();
+	sceKernelExitProcess(0);
```

Comment:
It  sometimes give an error message dialog when exit, and I have tested it again, but it exit normally. May because my old toolchains problem?